readme.md

```diff
@@ -16,6 +16,19 @@ c̶a̶r̶g̶o̶crunch clippy --workspace
 c̶a̶r̶g̶o̶crunch t -p sys-internals
 ```
 
+### Clean Builds with `--use-temp-dir`
+
+By default, `crunch` mirrors your local directory structure on the
+remote server. For an approach that avoids filesystem assumptions, at
+the cost of more bandwidth usage, go with the `--use-temp-dir` flag:
+
+```bash
+crunch --use-temp-dir build --release
+```
+
+This creates builds in temporary directories (e.g.,
+`/tmp/crunch-<project-name>-XXXXXX`).
+
```

Comment:
I prefer to keep flag docs in the doc comments, so it all ends up in the `--help` output.